### DISCUSSION TITLE
[Bug]: Formula on rollup

### DISCUSSION THREAD

**Discussion by arminta7** [RafaelGB/obsidian-db-folder#667] [Q&A]
### What happened?

I'm trying to perform addition on two values, one inline and the other a rollup. I'm not sure the number type is being carried over through the rollup because the addition simply concatenates the values. Also the rollup value style looks like a text field. 

This is my forumla: `${row.Stuff + row.Bar}`

<img width="791" alt="Screenshot 2022-12-13 at 19 04 11" src="https://user-images.githubusercontent.com/7927356/207471768-84c8163f-40d7-46b8-95bf-326cbebc4551.png">

### What platform were you using?

Desktop

### Version of the plugin

3.0.1

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hello @arminta7,
The type of those cells depend of how are saved. With rollups, formulas… are always text.

Try this to parse it into a number:

`${row.Stuff+Number(row.Bar)}`

**Comment by RafaelGB** [comment]
move to discussions as documentation
